scripts/buildChainsJson.js

```diff
@@ -0,0 +1,104 @@
+const path = require('path');
+const { writeFile } = require('fs/promises');
+const axios = require('axios');
+
+const NOVA_CONFIG_VERSION = process.env.NOVA_CONFIG_VERSION;
+const TELENOVA_CONFIG_VERSION = process.env.TELENOVA_CONFIG_VERSION;
+const CONFIG_PATH = `chains/${TELENOVA_CONFIG_VERSION}/`;
+const TELENOVA_CONFIG_URL = `https://raw.githubusercontent.com/novasamatech/nova-utils/master/chains/${NOVA_CONFIG_VERSION}/`;
+
+const CHAINS_ENV = ['chains_dev.json', 'chains.json'];
+const ALLOWED_CHAINS = require('./data/allowedChains.json');
+
+async function getDataViaHttp(url, filePath) {
+  try {
+    const response = await axios.get(url + filePath);
+
+    return response.data;
+  } catch (error) {
+    console.log('Error: ', error?.message || 'getDataViaHttp failed');
+  }
+}
+
+function fillAssetData(chain) {
+  const assetsList = [];
+  const allowedAssets = ALLOWED_CHAINS[chain.chainId]?.assets || [];
+
+  chain.assets.map(asset => {
+    const allowedAsset = allowedAssets.find(a => a.symbol === asset.symbol);
+    if (allowedAsset) {
+      assetsList.push({
+        assetId: asset.assetId,
+        symbol: asset.symbol,
+        precision: asset.precision,
+        type: asset.type,
```

Comment:
I want to suggest a special `asset type` - `native` for all native tokens
Because in code we have 2 actual types - `statemine` & `orml`.
But there is no type for native token, which leads conditions like `asset.type === undefined` in every place where  actions based on type are needed ☹️